README.md

```diff
@@ -1,70 +1,21 @@
-# Getting Started with Create React App
+### News-Explorer Frontend
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
+This is the frontend side of the News-Explorer final project
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## React
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+The frontend of the app was created with React, structuring components to render all parts of the app, using state variables and hooks to make the app functional.
+React router was used to route all components to render when needed and to protect components that should not be accessed
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+## Node.js
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Node.js is implemented to manage external libraries and for making scripts for quick executing of frequently used commands, such as building a production build for deployment (npm run build) and for starting the app in the local machine (npm run start).
 
-## Learn More
+## Links
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [Domain for the Front-End APP](www.yotamfinal.students.nomoredomainssbs.ru)
```

Comment:
` [Domain for the Front-End APP](www.yotamfinal.students.nomoredomainssbs.ru)`


you need to add the full link here to make it work (with `https://`)


```md

[Domain for the Front-End APP](https://www.yotamfinal.students.nomoredomainssbs.ru)



```

Reply:
fixed